api/specs.proto

```diff
@@ -206,14 +206,14 @@ message EndpointSpec {
 		// network. The user may then provide an alternate
 		// option for ingress load balancing which is
 		// transparent to swarm.
-		DISABLED = 1 [(gogoproto.enumvalue_customname) = "IngressRoutingDisabled"];
+		LOCAL = 1 [(gogoproto.enumvalue_customname) = "IngressRoutingDisabled"];
```

Comment:
@mrjana How can the haproxy (im assuming this is deployed via swarmkit?) get inbound traffic from something that lives outside of the ingress network? Example internet traffic...